04-Multitasking/os.c

```diff
@@ -63,6 +63,15 @@ unsigned int *create_task(unsigned int *stack, void (*start)(void))
 	return stack;
 }
 
+/* After reset, processor is at thread and privileged mode.
+ * Switch to handler mode to ensure an appropriate exception return.
+ */
+void task_init(void)
+{
+	unsigned int null_stacks[32];
```

Comment:
`null_stacks` is not an effective name. How about empty?

Reply:
Ok, this may apply to 06-Preemptive too.